android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -222,6 +222,10 @@ public void disconnectSession(Callback callback) {
         disconnectCallback = callback;
         if (mSession != null) {
             mSession.disconnect();
+            if (connectionStatus == 0) {
```

Comment:
The `disconnectCallback` is actually called here: https://github.com/opentok/opentok-react-native/pull/262/files#diff-5ddd727d527b5040249eccb8adc58ea5R438

But maybe we should dispatch it where you mention it.